app/code/core/Mage/Eav/Model/Adminhtml/System/Config/Source/Inputtype/Validator.php

```diff
@@ -22,7 +22,7 @@
 class Mage_Eav_Model_Adminhtml_System_Config_Source_Inputtype_Validator extends Zend_Validate_InArray
 {
     /**
-     * @var array<string, string>|null
+     * @inheritdoc
```

Comment:
Remove property?

Reply:
mmm it would rewrite some more rewrite because the base class initialise the array with something in it, the openmage does does it completely differently and uses a different message.